custom-plugins/fictive_studios/admin/printing-area/create/print-area-create.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace FictiveCodes;
+
+class PrintAreaCreate
+{
+    public function get_create_print_area_template()
+    {
+        $action = ($_GET['action']);    
+        $print_area_id;
```

Comment:
do we need this?

Reply:
I use those variables to retrieve print area data info based upon the url parameters if the page in in edit state.